Userland/Libraries/LibCrypto/Hash/SHA1.cpp

```diff
@@ -68,13 +70,16 @@ inline void SHA1::transform(u8 const* data)
 
 void SHA1::update(u8 const* message, size_t length)
```

Comment:
looks very similar `to update_buffer` in SHA2… which you definitely noticed. is it sensible to abstract that too? 🤔

Reply:
The SHA1 and SHA256-512 families of hashes have some small differences in implementation. Not going to work on that in this PR, but feel free to look at it and send in a PR yourself!